lib/commands/plugin-management.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const path = require('path');
+const fetch = require('node-fetch');
+const HttpsProxyAgent = require('https-proxy-agent');
+const url = require('url');
+const chalk = require('chalk');
+const _ = require('lodash');
+const CLI = require('../../lib/classes/CLI');
+const ServerlessError = require('../../lib/serverless-error');
+
+const cli = new CLI(undefined);
+
+module.exports = {
+  validate({ serviceDir }) {
+    if (!serviceDir) {
+      throw new ServerlessError(
+        'This command can only be run inside a service directory',
+        'MISSING_SERVICE_DIRECTORY'
+      );
+    }
+  },
+
+  getServerlessFilePath({ serviceDir, configurationFilename }) {
+    if (configurationFilename) {
+      return path.resolve(serviceDir, configurationFilename);
+    }
+    throw new ServerlessError(
+      'Could not find any serverless service definition file.',
+      'MISSING_SERVICE_CONFIGURATION_FILE'
+    );
+  },
+
+  async getPlugins() {
+    const endpoint = 'https://raw.githubusercontent.com/serverless/plugins/master/plugins.json';
+
+    // Use HTTPS Proxy (Optional)
+    const proxy =
+      process.env.proxy ||
+      process.env.HTTP_PROXY ||
+      process.env.http_proxy ||
+      process.env.HTTPS_PROXY ||
+      process.env.https_proxy;
+
+    const options = {};
+    if (proxy) {
+      // not relying on recommended WHATWG URL
+      // due to missing support for it in https-proxy-agent
+      // https://github.com/TooTallNate/node-https-proxy-agent/issues/117
+      options.agent = new HttpsProxyAgent(url.parse(proxy));
+    }
+
+    const result = await fetch(endpoint, options);
+    return result.json();
+  },
+
+  getPluginInfo(name) {
+    let pluginInfo;
+    if (name.startsWith('@')) {
+      pluginInfo = name.slice(1).split('@', 2);
+      pluginInfo[0] = `@${pluginInfo[0]}`;
+    } else {
+      pluginInfo = name.split('@', 2);
+    }
+    return pluginInfo;
+  },
+
+  async display(plugins) {
```

Comment:
I don't see this util being used anywhere (?)

Reply:
Removed 👍